Scraper Scripts/test/scraperTest.js

```diff
@@ -0,0 +1,159 @@
+const {expect} = require("chai");
+const scraper = require("../scraper.js");
+
+describe("Web scraper unit tests", async () => {
+    // Test Steam getting a price from a paid game
+    it("should return a price from the SteamAPI", async () => {
+        var price = await scraper.GetSteam("2807960");
+        expect(price).to.notEqual(null);
+        expect(price).to.notEqual(0);
+    });
+
+    // Test Steam getting a free game
+    it("should return free from the SteamAPI", async () => {
+        var price = await scraper.GetSteam("730");
+        expect(price).to.equal(0);
+    });
+
+    // Test Epic getting price data from a paid game
+    it("should return a price from EGS", async () => {
+        var gameData = await scraper.GetEpic("Satisfactory");
+        expect(gameData).to.equal(!null);
+        expect(gameData["Title"]).to.equal("Satisfactory");
+        expect(gameData["OriginalPrice"]).to.equal(51.99);
+        expect(gameData["DiscountPrice"]).to.notEqual(null);
```

Comment:
this is not valid chai syntax
```suggestion
        expect(gameData["DiscountPrice"]).to.not.equal(null);
```